core/stat/stat_slot_callback.go

```diff
@@ -0,0 +1,75 @@
+package stat
+
+import (
+	"github.com/alibaba/sentinel-golang/core/base"
+	"sync"
+)
+
+type CallbackKey string
+
+const (
+	FreqParamsTraffic CallbackKey = "CallbackKey##FreqParamsTraffic"
+)
+
+var (
+	// global StatSlot callback
+	entryCallBackMap  = make(map[CallbackKey][]StatSlotEntryCallback)
+	exitCallBackMap   = make(map[CallbackKey][]StatSlotExitCallback)
+	callbackUpdateMux = new(sync.RWMutex)
+)
+
+type StatSlotEntryCallback interface {
```

Comment:
Maybe the stat slot callback is not needed? Just make it as another `StatSlot` and add it to slot chain is okay :)

Reply:
Adding a new StatSlot to record the concurrency statistic seems make more sense. I will reimplement related logic.